pyproject.toml

```diff
@@ -2,7 +2,7 @@
 # See LICENSE file for licensing details.
 
 [tool.bandit]
-exclude_dirs = ["/venv/"]
+exclude_dirs = ["/venv/", "tests/integration/any_charm.py"]
```

Comment:
Shouldn't the tests for any charm be in a file that starts with `test_`? That way the skip on line 7 should include it and we don't need it here I think.

Reply:
any-charm will probably be used in most integration tests as a target for content-cache. I could eventually write the whole content of the file into `conftest.py` to avoid this exclude but it seems cleaner that way for me.